authorization-dotnet/BouncyCastleCryptography/BouncyCastleCryptographyTests/SymmetricEncryptionUnitTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using BouncyCastleCryptography.SymmetricEncryption;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Security;
+
+namespace BouncyCastleCryptographyTests
+{
+    [TestClass]
+    public class SymmetricEncryptionUnitTests
+    {
+        [TestMethod]
+        public void GivenSecretData_WhenAesEncrypting_ThenDataIsEncrypted()
+        {
+            string input = "Hello, Bouncy Castle!";
+
+            var encryptedData = AesEncryptor.AesEncrypt(input, out byte[] iv, out byte[] key);
+
+            string encryptedString = Convert.ToBase64String(encryptedData);
+
+            Assert.IsNotNull(encryptedData);
+            Assert.AreNotEqual(input, encryptedString);
+        }
+
+        [TestMethod]
+        public void GivenSecretData_WhenAesEncrypting_ThenDataCanBeDecrypted()
+        {
+            string input = "Hello, Bouncy Castle!";
+
+            var encryptedData = AesEncryptor.AesEncrypt(input, out byte[] iv, out byte[] key);
+
+            Assert.IsNotNull(encryptedData);
+
+            var myDecryptedStr = AesEncryptor.AesDecrypt(key, iv, encryptedData);
+
+            Assert.IsNotNull(myDecryptedStr);
+            Assert.AreEqual(input, myDecryptedStr);
+        }
+
+        [TestMethod]
+        public void GivenSecretData_WhenTripleDesEncrypting_ThenDataIsEncrypted()
+        {
+            string input = "Hello, Bouncy Castle!";
+
+            var myStr = TripleDesEncryptor.TripleDesEncrypt(input, out byte[] iv, out byte[] key);
+
+            Assert.IsNotNull(myStr);
```

Comment:
For consistency, we should check the input data is not equals to the encrypted data here.